### PR TITLE
Post nominated notification

### PR DESCRIPTION
**Pull request by Raemon**
When someone votes on a post giving it 2+ positiveReviewVoteCount, send a notification:

image

and an email:

image

### CODE REVIEW

**Comment by Raemon** [comment]
I've made it so all email titles are the same (instead of having unique ones based on post-title) out of a vague hope that email clients will err towards collapsing them into a single thread, to avoid spamming popular authors. (Ideally I'd write a "batch" system, but there isn't the time)

File: packages/lesswrong/lib/collections/users/custom_fields.ts [hunk 1]
```diff
@@ -693,6 +693,10 @@ addFieldsDict(Users, {
     hidden: !hasEventsSetting.get(),
     ...notificationTypeSettingsField({ channel: "both" }),
   },
+  notificationPostsNominatedReview: {
```
Comment (jimrandomh) [inline packages/lesswrong/lib/collections/users/custom_fields.ts:696]: You want to also add this to the `UsersCurrent` fragment

Comment (Raemon) [reply packages/lesswrong/lib/collections/users/custom_fields.ts:696]: UsersCurrent or UsersEdit? Edit seems to be where the existing settings live

File: packages/lesswrong/server/emailComponents/PostNominatedEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const nominationEndDate = moment.utc(annualReviewNominationPhaseEnd.get())
+
+  return (<React.Fragment>
+    <p>Your post <a href={postGetPageUrl(post)}>{post.title}</a> has received multiple positive votes for the {REVIEW_NAME_IN_SITU}! On {nominationEndDate.format('MMM Do')}, the preliminary vote results will be published, and will be used to help prioritize the Review Phase.</p>
+    
+    <p>You're encouraged to write a self-review, exploring how you think about the post today. Do you still endorse it? Have you learned anything new that adds more depth? How might you improve the post? What further work do you think should be done exploring the ideas here?</p>
+    
+    <p>You can write a self-review by going to <a href={postGetPageUrl(post)}>your post</a> and clicking the "Write a Review" button.</p>
+
+    <p>You can see a list of nominated posts over at <a href="lesswrong.com/reviewVoting">the Review Dashboard page</a>. Posts with at least one review are sorted to the top.</p>
```
Comment (jpaddison3) [inline packages/lesswrong/server/emailComponents/PostNominatedEmail.tsx:39]: 👀

**Pull request merged**